libgalois/include/katana/PODVector.h

```diff
@@ -100,19 +124,27 @@ class PODVector {
   //! move assignment operator
   PODVector& operator=(PODVector&& v) {
     if (data_ != NULL) {
+      if (pinned_) {
+        munlock(data_, capacity_ * sizeof(_Tp));
+      }
       free(data_);
     }
     data_ = v.data_;
     capacity_ = v.capacity_;
     size_ = v.size_;
+    pinned_ = v.pinned_;
     v.data_ = NULL;
     v.capacity_ = 0;
     v.size_ = 0;
+    v.pinned_ = false;
```

Comment:
This duplicates the move constructor (and the default constructor if that exists). Maybe factor it out into a private `Clear` method.